### PR TITLE
store: make sure expectedZeroFields in tests gets updated

### PR DESCRIPTION
**Pull request by pedronis**
drop Assumes

change the test so that the case where a field is now set gets noticed

### CODE REVIEW

**Comment by pedronis** [comment]
small follow-up to #10659 and #10660

File: store/details_v2_test.go [hunk 1]
```diff
@@ -288,7 +288,6 @@ func (s *detailsV2Suite) TestInfoFromStoreSnap(c *C) {
 	expectedZeroFields := []string{
 		"SuggestedName",
 		"InstanceKey",
-		"Assumes",
```
Comment (mvo5) [inline store/details_v2_test.go:290]: \o/ Thanks for tihs

**Review by mvo5 — APPROVED** [review]
Thank you (looking at the last commit)

File: store/details_v2_test.go [hunk 2]
```diff
@@ -324,9 +323,11 @@ func (s *detailsV2Suite) TestInfoFromStoreSnap(c *C) {
 				checker(pfx+f.Name+".", v)
 				continue
 			}
+			name := pfx + f.Name
 			if reflect.DeepEqual(v.Interface(), reflect.Zero(f.Type).Interface()) {
-				name := pfx + f.Name
 				c.Check(expectedZeroFields, testutil.Contains, name, Commentf("%s not set", name))
+			} else {
```
Comment (anonymouse64) [inline store/details_v2_test.go:329]: thanks for adding this additional check, hopefully it should make it more obvious in the future

**Review by anonymouse64 — APPROVED** [review]
lgtm, thanks for the additional test case too

**Pull request merged**